### DISCUSSION TITLE
Error while running PhpStan / Larastan: Call to method newScreen() on an unknown class    Spatie\WordPressRay\Ray.

### DISCUSSION THREAD

**Discussion by fabioi** [spatie/laravel-ray#96] [Q&A]
"php": "^7.4",
spatie/laravel-ray: 1.5.2
nunomaduro/larastan": v0.6.12

Hello, first of all, thank you a lot for your package! Super helpful and fun to use!

Just wanted to signal that PhpStan seems to have trouble understanding ray.

```php
    public function testItGetsTheCreatePage(): void
    {
        ray()->newScreen('testItGetsTheCreatePage');

        $response = $this->actingAs($this->getStaffUser())->get(route('backend.users.create'));

        $response->assertViewHas('allRoles', Role::all()->pluck('id', 'name')->toArray());
        $response->assertViewHas('action', route('backend.users.store'));

        $response->assertSuccessful();
    }

```

phpstan
=======

------ --------------------------------------------------------------------
  Line   Modules/User/Tests/Feature/Backend/ItCanCreateANewUserTest.php
 ------ --------------------------------------------------------------------
  20     Call to method newScreen() on an unknown class
         Spatie\WordPressRay\Ray.
         💡 Learn more at https://phpstan.org/user-guide/discovering-symbols
  32     Call to method newScreen() on an unknown class
         Spatie\WordPressRay\Ray.
         💡 Learn more at https://phpstan.org/user-guide/discovering-symbols
 ------ --------------------------------------------------------------------

**Comment by freekmurze** [comment]
It might be best to configure phpstan to remove that warning for now, as we won't be making any changes our this anytime soon.

**Reply by fabioi** [reply]
done, thank you!

**Comment by Patabugen** [comment]
In case it helps anyone else, here's how I disabled these errors.

It took me a few attempts before I noticed the full stop at the end of the message.

In `phpstan.neon.dist`

```YAML
parameters:
    ignoreErrors:
        - '#^Call to method .+ on an unknown class Spatie\\.+\\Ray.$#'
```
